compiler/quilt/tools/store.py

```diff
@@ -13,6 +13,7 @@
 
 # start with alpha (_ may clobber attrs), continue with alphanumeric or _
 VALID_NAME_RE = re.compile(r'^[a-zA-Z]\w*$')
+VALID_EXTENDED_PCKG_RE = re.compile(r'[\w+:]?\w+/\w+:(version|tag|hash|v|t|h):(.+)', flags=re.IGNORECASE)
```

Comment:
The `[\w+:]` doesn't seem right; did you mean `(\w+:)`?

Reply:
Also, probably need `^` and `$`

Reply:
I just didn't want to extract team, but I  guess it's completely fine, just ignore this group.

Reply:
I just didn't want to extract team, but I guess it is fine to ignore it.